htmresearch/support/neural_correlations_utils.py

```diff
@@ -153,7 +153,7 @@ def accuracy(current, predicted):
   """  
   acc = 0
   if np.count_nonzero(predicted) > 0:
-    acc = float(np.dot(current, predicted))/float(np.count_nonzero(predicted))
+    acc = float(np.dot(current, predicted))/float(np.count_nonzero(current))
```

Comment:
ok. Let me ask Subutai, because I thought that we should also consider the cases when the TM did correct predictions even among multiple predictions.